db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 2020_07_30_194003) do
+ActiveRecord::Schema.define(version: 2020_06_01_204556) do
```

Comment:
Is this rollback intentional?

Reply:
reverted

Reply:
I see the version was reverted but `is_public` is still being removed from the accounts table.